### PR TITLE
[SYCL] Temporarily turning off copy engine support for memory fill operations

### PR DESCRIPTION
**Pull request by asudarsa**
Signed-off-by: Arvind Sudarsanam <arvind.sudarsanam@intel.com>

### CODE REVIEW

File: sycl/plugins/level_zero/pi_level_zero.cpp [hunk 1]
```diff
@@ -4957,6 +4957,9 @@ enqueueMemFillHelper(pi_command_type CommandType, pi_queue Queue, void *Ptr,
   ze_command_list_handle_t ZeCommandList = nullptr;
   ze_fence_handle_t ZeFence = nullptr;
 
+  // TODO: Fill operations on copy engine fails to fill a buffer at expected
+  // offset. Perform fill operations on compute engine for now.
+  // PreferCopyEngine will be initialized with 'true' once issue is resolved.
```
Comment (smaslov-intel) [inline sycl/plugins/level_zero/pi_level_zero.cpp:4962]: If we turn it to true here then the check for max pattern size needs to be able to set it false. So, probably, invert the condition there.

Comment (asudarsa) [reply sycl/plugins/level_zero/pi_level_zero.cpp:4962]: That's a good point. Done.

**Review by smaslov-intel — APPROVED** [review]

**Comment by bader** [comment]
The fact that none of pre-commit tests failed for this change concerns me.

**Comment by asudarsa** [comment]
Hi @bader , Thanks for merging this. I am in the process of adding a set of regression tests for blitter support. This has got delayed due to my pulldown co-ordinator responsibilities. I will get to adding these tests during the first week of June.

**Pull request merged**